Cargo.toml

```diff
@@ -38,6 +38,7 @@ prometheus_exporter = ["metrics-exporter-prometheus", "metrics"]
 
 # state features
 rocksdb = ["arcon_state/rocks"]
+arcon_allocator_counters = ["arcon_allocator/allocator_metrics"]
```

Comment:
arcon_allocator_counters -> allocator_metrics = ["arcon_allocator/metrics"]

And let's move it above the state features comment in order to not cause confusion.